tasks/deployment_manager/task.ts

```diff
@@ -152,5 +154,9 @@ task('migrate', 'Runs migration')
       }
 
       await runMigration(dm, prepare, enact, migration, overwrite);
+
+      if (enact && setEnacted) {
```

Comment:
Was suggesting `noEnact` so it would default enable, you think it makes more sense to default disable?

Reply:
Good point, converted to `noEnacted`